### DISCUSSION TITLE
is it possible to set svg image as joint js diagram background image

### DISCUSSION THREAD

**Discussion by reshin-pulapaddy** [clientIO/joint#1938] [Q&A]
is it possible to set SVG image as a joint js diagram background image?

**Comment by jamesgeorgewilliams** [comment]
You can set an image as the background with the paper `background` option:
https://resources.jointjs.com/docs/jointjs/v3.6/joint.html#dia.Paper.prototype.options.background

```js
const paper = new joint.dia.Paper({
    background: {
         image: 'path/to/your-image.svg',
         opacity: 0.3,
         size: { width: 200, height: 200 }
    }
});
```

**Reply by reshin-pulapaddy** [reply]
thank you @jamesgeorgewilliams
